myRental/modules/database.js

```diff
@@ -44,6 +29,7 @@ method.registerUser = (request, callBack) => {
             });
         })
         .catch(function (err) {
+            console.log(err);
```

Comment:
In the future, it may help to set up log files

Reply:
That could be an easy story to add for the next meeting. Might give something like that to Ben to get him introduced into the back end.